setup.md

```diff
@@ -4,8 +4,15 @@ title: Setup
 root: .
 ---
 
-You need to have the following software installed on your computer to
-follow this lesson.
+For this lesson, you need to have 
+[Bash](http://swcarpentry.github.io/shell-novice), 
+[Git](http://swcarpentry.github.io/git-novice), 
+[Python](http://swcarpentry.github.io/python-novice-inflammation), 
+[Docker](https://ome.github.io/training-docker/) and Popper installed 
+on your system. If you have completed the [Software Carpentry 
+lesson](https://software-carpentry.org/) for any of these, you do not 
+need to do the setup again. Below, you will find instructions on how 
+to install each of these tools.
 
```

Comment:
Maybe make a more clear distinction between Popper itself and the prerequsites?

Reply:
thanks for the feedback @alex-konovalov, I've made a change to distinguish between popper setup, do you think that would work?